CONTRIBUTING.md

```diff
@@ -238,40 +238,39 @@ Changes for a given release should be split between the five sections:
 You must have release-me installed and configured with a token. See
 https://pypi.org/project/release-me/
 
-Assuming the version to be released is `l.m.n`, as per semantic versioning, the
-current release process is as follows:
+Assuming the current version recorded in the project's `.pom` files is
+`l.m.n-SNAPSHOT`, the manual release process is as follows:
 
 ### Prepare the release
 
-- [ ] Create a new feature branch, `release-l.m.n` and check it out
-- [ ] Generate the release notes by running `./script/release-notes.sh`, this
-      will create a file `RELEASE-NOTES.md`.
-- [ ] Mark the version as RELEASE via `mvn versions:set -DnewVersion=l.m.n-RELEASE`
-- [ ] Commit the changes: `git add . && git commit -m "Prepare for release l.m.n"`
-- [ ] Open a PR merging the feature branch into `develop`. Get it merged.
-- [ ] Open a PR to merge `unstable` into `master`, titling it `Release l.m.n`
+- [ ] `git checkout unstable && git pull`
+- [ ] Run `./script/release-prepare.sh` to
+  - create and checkout a branch `release/l.m.n`.
+  - prepare an commit a release commit `[release] l.m.n`
+- [ ] Run `./script/version-bump.sh` to
+  - update the changelog
+  - bump the version number
+  - commit the changes
+- [ ] Open a PR merging the newly created branch into `unstable`, with the title
+      `Release l.m.n`.
+- [ ] Get the PR reviewed and merged and **DO NOT SQUASH THE CHANGES** on merge.
+
+If you need to set a specific version (e.g., to increment to a major version),
+override the `RELEASE_VERSION` when preparing the release:
+
+```sh
+RELEASE_VERSION=l.m.n ./script/release-prepare.sh
+```
 
 ### Cut the release
 
-When the PR is merged into `master`:
+When the PR is merged into `unstable`:
```

Comment:
This will be done by CI usually, and I'll update the docs here once that's implemented.